### PR TITLE
Implement quick reply

### PR DESCRIPTION
**Pull request by be-hase**
https://github.com/line/line-bot-sdk-python/issues/119

### CODE REVIEW

File: linebot/models/__init__.py [hunk 1]
```diff
@@ -47,6 +50,23 @@
     Beacon,
     Link,
 )
+from .flex_message import (  # noqa
```
Comment (be-hase) [inline linebot/models/__init__.py]: Changed in alphabet order

File: examples/flask-kitchensink/app.py [hunk 2]
```diff
@@ -278,6 +280,34 @@ def handle_text_message(event):
...
+                        QuickReplyButton(
+                            action=CameraAction(label="label4")
+                        ),
+                        QuickReplyButton(
+                            action=CameraRollAction(label="label5")
+                        ),
+                        QuickReplyButton(
+                            action=LocationAction(label="label6")
+                        ),
+                    ])))
```
[the remark this answers is not in the document]
Comment (be-hase) [reply examples/flask-kitchensink/app.py]: @Sato-Kunihiko 
Ah... I don't distinguish it for a clear reason.
if I’m forced to say, I often use single quotes for small symbol-like strings.

**Comment by be-hase** [comment]
Thanks :)

**Comment by DongShanHu** [comment]
Hello , I want to ask some questions
when I want to test quick reply function, I use "from linebot.model import * ,and follow your step to do, but the error is the quick reply is not defined......???

**Comment by be-hase** [comment]
@DongShanHu 
This quick reply feature is not released yet. please wait.

**Pull request merged**